### PR TITLE
Minor code changes, mostly cleanup

### PR DESCRIPTION
**Pull request by pnmtjonahen**

### CODE REVIEW

File: model.js [hunk 1]
```diff
@@ -27,7 +27,6 @@ module.exports = {
    }
 };
 var http             = require("http"),
-    url              = require("url"),
```
Comment (IvoNet) [inline model.js:30]: why?

File: scooch.js [hunk 2]
```diff
@@ -43,6 +43,20 @@ function startsWith(str, postfix) {
    return str.indexOf(postfix, 0) === 0;
 }
 
+function sendContentNotFound(filename, response) {
+   console.log(filename + " resource not found.");
+   response.writeHead(404, {"Content-Type": "text/plain"});
```
Comment (IvoNet) [inline scooch.js:48]: Why text/plain en niet html?

Comment (pnmtjonahen) [inline scooch.js:48]: Ok, will change

File: model.js (hunk 1 above)
Comment (pnmtjonahen) [inline model.js:30]: Because

**Pull request merged**